### DISCUSSION TITLE
spdlog with fmt 9.0.0?

### DISCUSSION THREAD

**Discussion by graygoesblack** [gabime/spdlog#2455] [General]
Hi All,

I'm pretty new to spdlog and was wondering if it's possible to use it with newest version of fmt, 9.0.0? If I try to build latest spdlog with it (externally) there are many linker errors. Or do I need to edit configure files in order to run it?

**Comment by tt4g** [comment]
spdlog is not supported with fmt 9.0.0 (the bundled fmt version is still 8.1.1).

**Comment by coreyhartley** [comment]
I'm not sure if this is related to the original question, but my build all of a sudden is throwing a handful of `fmt::v9`...`was declared deprecated` errors.  I'm pulling spdlog into my project using vcpkg, so I'm not sure if it's pulling the wrong version of fmt through there or what?

**Reply by tt4g** [reply]
That question should be asked in the vcpkg project, vcpkg may support spdlog with fmt v9, spdlog is not yet supported.

**Reply by coreyhartley** [reply]
Yeah, I'll do that.  From what I can see in my vcpkg files, it looks like the only package that uses fmt is spdlog, so I'm assuming it's an issue with pulling and using the correct version of the dependency.  Thought I would post here to see if anyone else has had the issue, and for awareness.
